lib/eth/EthRpc.js

```diff
@@ -204,8 +204,15 @@ class EthRPC {
     return this.web3.eth.personal.lockAccount(account);
   }
 
-  getTransaction({ txid }) {
-    return this.web3.eth.getTransaction(txid);
+  async getTransaction({ txid, detail = false }) {
+    const tx = await this.web3.eth.getTransaction(txid);
+    if (!detail || !tx) {
+      return tx;
+    }
+    const bestBlock = await this.web3.eth.getBlockNumber();
+    const confirmations = tx.blockNumber ? bestBlock - tx.blockNumber + 1 : 0;
```

Comment:
This would return 0 confirmations for transactions in the genesis block I think. May want an explicit undefined check instead

Reply:
indeed